java/src/jmri/jmrit/beantable/LogixTableAction.java

```diff
@@ -1202,7 +1202,7 @@ void showSaveReminder() {
         if (_showReminder) {
             if (InstanceManager.getDefault(jmri.UserPreferencesManager.class) != null) {
                 InstanceManager.getDefault(jmri.UserPreferencesManager.class).
-                        showInfoMessage(rbx.getString("ReminderTitle"), rbx.getString("Reminder1"), getClassName(), "remindSaveLogix");
+                        showInfoMessage(Bundle.getMessage("ReminderTitle"), Bundle.getMessage("ReminderSaveString", Bundle.getMessage("MenuItemLogixTable")), getClassName(), "remindSaveLogix");
```

Comment:
This needs a ```// NOI18N``` marker to indicate that these String literals do not need translation.